### ISSUE TITLE
LibOParl compilation failed on macOS with valac 0.36.1

### ISSUE THREAD

**Issue by eFrane** [OParl/liboparl#13]
```bash
graupner@felicity:build> meson
The Meson build system
Version: 0.39.1
Source dir: /Users/sgraupner/Development/OParl/liboparl
Build dir: /Users/sgraupner/Development/OParl/liboparl/build
Build type: native build
Project name: oparl
Native vala compiler: valac (valac 0.36.1)
Native c compiler: cc (clang 8.1.0)
Build machine cpu family: x86_64
Build machine cpu: x86_64
Found pkg-config: /usr/local/bin/pkg-config (0.29.2)
Native dependency glib-2.0 found: YES 2.52.0
Native dependency gobject-2.0 found: YES 2.52.0
Native dependency json-glib-1.0 found: YES 1.2.8
Native dependency gio-2.0 found: YES 2.52.0
Program g-ir-compiler found: YES (/usr/local/bin/g-ir-compiler)
Program valadoc found: YES (/usr/local/bin/valadoc)
Build targets in project: 4
sgraupner@felicity:build> ninja
[1/36] 'Generating apidocs with a custom command.'
FAILED: devhelp
'/usr/local/bin/valadoc' '-o' 'devhelp/oparl-0.2' '--doclet' 'devhelp' '../src/oparl.vala' '../src/body.vala' '../src/person.vala' '../src/membership.vala' '../src/meeting.vala' '../src/system.vala' '../src/object.vala' '../src/organization.vala' '../src/agenda_item.vala' '../src/paper.vala' '../src/consultation.vala' '../src/legislative_term.vala' '../src/location.vala' '../src/file.vala' '--pkg' 'json-glib-1.0' '--force'
error: failed to load driver
Failed: 1 error(s), 0 warning(s)
[2/36] Compiling Vala source ../src/oparl.vala ../src/body.vala ../src/person.vala ...nsultation.vala ../src/legislative_term.vala ../src/location.vala ../src/file.vala.
FAILED: oparl-0.2@sha/oparl.c oparl-0.2@sha/body.c oparl-0.2@sha/person.c oparl-0.2@sha/membership.c oparl-0.2@sha/meeting.c oparl-0.2@sha/system.c oparl-0.2@sha/object.c oparl-0.2@sha/organization.c oparl-0.2@sha/agenda_item.c oparl-0.2@sha/paper.c oparl-0.2@sha/consultation.c oparl-0.2@sha/legislative_term.c oparl-0.2@sha/location.c oparl-0.2@sha/file.c oparl-0.2.h oparl-0.2.vapi
valac '--debug' '-d' 'oparl-0.2@sha' '-C' '--library=oparl-0.2' '-H' 'oparl-0.2.h' '--vapi' '../oparl-0.2.vapi' '--pkg' 'glib-2.0' '--pkg' 'gobject-2.0' '--pkg' 'gio-2.0' '--pkg' 'json-glib-1.0' '--gir=../OParl-0.2.gir' ../src/oparl.vala ../src/body.vala ../src/person.vala ../src/membership.vala ../src/meeting.vala ../src/system.vala ../src/object.vala ../src/organization.vala ../src/agenda_item.vala ../src/paper.vala ../src/consultation.vala ../src/legislative_term.vala ../src/location.vala ../src/file.vala
../src/object.vala:135.43-140.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/object.vala:356.48-362.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/object.vala:365.48-371.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied


  1 LibOParl compilation failed on macOS with valac 0.36.1
../src/agenda_item.vala:229.48-234.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/file.vala:270.48-275.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/file.vala:278.48-284.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/meeting.vala:285.48-290.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/membership.vala:187.48-192.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/organization.vala:272.48-277.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/legislative_term.vala:104.48-109.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:395.48-401.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:404.48-409.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:412.48-418.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:421.48-426.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:429.48-435.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:438.48-443.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:446.48-452.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:455.48-460.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:463.48-469.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:472.48-477.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/body.vala:480.48-485.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/system.vala:175.48-180.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/system.vala:184.48-189.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/system.vala:192.48-197.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/system.vala:200.48-205.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/object.vala:374.48-381.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/object.vala:388.52-393.21: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/object.vala:396.52-403.21: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/object.vala:407.48-413.17: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/object.vala:424.52-431.21: error: Access to non-public constructor `OParl.ValidationResult.new' denied
../src/object.vala:434.52-441.21: error: Access to non-public constructor `OParl.ValidationResult.new' denied
Compilation failed: 31 error(s), 0 warning(s)
ninja: build stopped: subcommand failed.
1 sgraupner@felicity:build> valac --version
Vala 0.36.1
```

**Comment by grindhold** [comment]
still waiting for vala 0.36.1 on debian testing

**Comment by eFrane** [comment]
that's still unfortunate

**Comment by grindhold** [comment]
solved as of bf44b1d
